problemsets-mat223/modules/module8-exercises.tex

```diff
@@ -1,17 +1,53 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
+
+		\prob
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
+
+			\item Let $\vec u=\xhat+8\yhat$, $\vec v=-\xhat+ 3\yhat$, and $\vec w=2\xhat$.
+
+			\begin{enumerate}
+				\item Find $[\vec u]_{\mathcal E}$, $[\vec v]_{\mathcal E}$ and
+				$[\vec w]_{\mathcal E}$ where $\mathcal E$ is the standard basis for $\R^2$.
+				\item Let $\mathcal A=\Set{3\xhat+2\yhat,4\xhat-\yhat}$.
+				Find $[\vec u]_{\mathcal A}$, $[\vec v]_{\mathcal A}$ and
+				$[\vec w]_{\mathcal A}$.
+				\item Let $\mathcal B=\Set{11\yhat,\xhat+\frac{5}{2}\yhat}$.
+				Find $[\vec u]_{\mathcal B}$, $[\vec v]_{\mathcal B}$ and
+				$[\vec w]_{\mathcal B}$.
+			\end{enumerate}
+
+			\item Let $\vec q=11\yhat-4\zhat$, $\vec r=5\xhat-12\yhat+8\zhat$, and $\vec s=\xhat+\zhat$.
+
 			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
+				\item Find $[\vec q]_{\mathcal E}$, $[\vec r]_{\mathcal E}$ and
+				$[\vec s]_{\mathcal E}$ where $\mathcal E$ is the standard basis for $\R^3$.
+				\item Let $\mathcal D=\Set{1\xhat+2\yhat,-3\xhat+5\yhat-4\zhat,-8\xhat+4\yhat+11\zhat}$.
+				Find $[\vec q]_{\mathcal D}$, $[\vec r]_{\mathcal D}$ and
+				$[\vec s]_{\mathcal D}$.
+				\item Let $\mathcal F=\Set{1\xhat+4\yhat+4\zhat,-3\xhat+20\yhat,21\yhat+16\zhat}$.
+				Find $[\vec q]_{\mathcal F}$, $[\vec r]_{\mathcal F}$ and
+				$[\vec s]_{\mathcal F}$.
 			\end{enumerate}
-		\end{solution}
+		\end{enumerate}
+
+		\prob
+		\begin{enumerate}
+			\item Let $[\vec a]_{\mathcal E}=\mat{5\\-12}$ where $\mathcal E$ is the standard basis
+			for $\R^2$. Find a basis $\alpha$ for $\R^2$ such that $[\vec a]_{\alpha}=\mat{1\\0}$.
```

Comment:
We name our bases with capital roman letters in this class (greek letters are reserved for scalars)